honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,173 @@
+                        'condition and a True is_operable property.'
+        )
+
+
+class DoorModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Doors."""
+
+    type: constr(regex='^DoorModifierSet$') = 'DoorModifierSet'
+
+    exterior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for doors with an Outdoors '
+                        'boundary condition.'
+        )
+
+    interior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for doors with a Surface '
+                        'boundary condition.'
+        )
+
+    exterior_glass_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for glass with an Outdoors '
+                        'boundary condition.'
+        )
+
+    interior_glass_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A modifier object for glass with a Surface '
+                        'boundary condition.'
+        )
+
+    overhead_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A window modifier object for doors with an Outdoors boundary '
+                        'condition and a RoofCeiling or Floor face type for their '
+                        'parent face.'
+        )
+
+
+class ModifierSet(IDdBaseModel):
+    """Set containing all radiance modifiers needed to create a radiance model."""
+
+    type: constr(regex='^ModifierSet$') = 'ModifierSet'
+
+    wall_set: WallModifierSet = Field(
+        default=None,
+        description='An optional WallModifierSet object for this ModifierSet. '
+                    '(default: None).'
+    )
+
+    floor_set: FloorModifierSet = Field(
+        default=None,
+        description='An optional FloorModifierSet object for this ModifierSet. '
+                    '(default: None).'
+    )
+
+    roof_ceiling_set: RoofCeilingModifierSet = Field(
+        default=None,
+        description='An optional RoofCeilingModifierSet object for this ModifierSet. '
+                    '(default: None).'
+    )
+
+    aperture_set: ApertureModifierSet = Field(
+        default=None,
+        description='An optional ApertureModifierSet object for this ModifierSet. '
+                    '(default: None).'
+    )
+
+    door_set: DoorModifierSet = Field(
+        default=None,
+        description='An optional DoorModifierSet object for this ModifierSet. '
+                    '(default: None).'
+    )
+
+    shade_set: ShadeModifierSet = Field(
+        default=None,
+        description='An optional ShadeModifierSet object for this ModifierSet. '
+                    '(default: None).'
+    )
+
+    # TODO: This requires default modfiers from honeybee-radiance? How to schematize?
+    # air_boundary_modifier: AirWallModifier = Field(...)
```

Comment:
The `air_boundary_modifier` is just like any other slot for a modifier. It should be:
```
Union[Plastic, Glass, BSDF, Glow, Light, Trans]
```